src/security/lockdown/Kconfig

```diff
@@ -62,7 +62,6 @@ config BOOTMEDIA_LOCK_WHOLE_NO_ACCESS
 
 config BOOTMEDIA_LOCK_WPRO_VBOOT_RO
 	bool "Write-protect WP_RO FMAP region in boot medium"
-	depends on VBOOT
```

Comment:
Why is this needed? Won't `BOOTMEDIA_LOCK_WHOLE_RO` with the old fmap do what we want here? I'm assuming the point is to write-protect COREBOOT and FMAP without enabling Vboot entirely